### DISCUSSION TITLE
Filter warning

### DISCUSSION THREAD

**Discussion by getrude386** [mboquien/cigale#14] [Q&A]
Hi, can you please help me, how do I solve this:
Warning: CFHT_g in the input file but not to be taken into account in the fit.
Warning: CFHT_r in the input file but not to be taken into account in the fit.

It's a list of many bands (just picked two),  only the following bands were fitted:

bands = L, L_err, CFHT_u, J, H, CFHT_u_err, J_err, H_err

**Comment by mboquien** [comment]
This is not an error, just a warning that the g and r CFHT bands are in the input file but they are not fitted. If it is what you intend, you can safely ignore this warning. It is there just in case the user actually wanted to include these bands but forgot.

**Reply by getrude386** [reply]
If all my bands are not fitted, my results won't be affected?

**Reply by mboquien** [reply]
They will, naturally. But that is the choice of the user to decide what bands to fit or not to fit.

**Reply by getrude386** [reply]
![Screenshot 2022-06-28 at 13 22 32](https://user-images.githubusercontent.com/105718721/176170486-5bff19e4-c7cc-41e4-969f-5f48d42cf429.png)

Not all the models were plotted.

**Reply by getrude386** [reply]
> They will, naturally. But that is the choice of the user to decide what bands to fit or not to fit.

I actually wanted to fit all the bands contained in my catalogue.

**Reply by mboquien** [reply]
You can remove the list of bands before running `pcigale genconf`. It will take all the columns whose names correspond to bands in the filters database. You can get the full list with `pcigale-filters list`. To get all the CFHT bands you can do `pcigale-filters list | grep -i cfht` for instance. Then you just need to adjust the column names in you input file before running `pcigale genconf`.

**Reply by getrude386** [reply]
Ok I get it now, thank you.

**Reply by getrude386** [reply]
Hi, please help me understand this (attached image)
![Screenshot 2022-06-29 at 19 52 36](https://user-images.githubusercontent.com/105718721/177106693-921e542d-84ff-49ff-9eef-3148a9337e27.png)
, I get it when running: pcigale-plots sed, out of 18 sources, only 11 seds get plotted.


Also, out of 18 sources, cigale found only 16 models. Is that possible?

**Reply by mboquien** [reply]
It is because it could not find a best fit. Typically that happens when there is only one valid band or that the fits are so catastrophically bad that the likelihood underflows to zero.

**Reply by getrude386** [reply]
Is that why I am getting the kind of results?
![Screenshot 2022-07-04 at 09 17 45](https://user-images.githubusercontent.com/105718721/177148847-2e862b0a-2f9d-4563-b927-5c53342f0c2e.png)

**Reply by mboquien** [reply]
Yes. Your photometry seems problematic.

**Reply by getrude386** [reply]
I will check it, thank you.
